### PR TITLE
modfied the file to fix bug

### PR DESCRIPTION
**Pull request by mayankbharadwaj**
Purpose:

Bug Fix MA-4034

Changes:

-  Modified iam.js

### CODE REVIEW

File: aws_promise/iam.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
-'use strict'
+'use strict';
 
 const AWS = require('aws-sdk');
 
 module.exports = {
 
   findService: function(input) {
-    const iam;
+    let iam;
```
Comment (SonalOjha) [inline aws_promise/iam.js:8]: whats causing the issue?

**Comment by SonalOjha** [comment]
do not merge into develop

File: aws_promise/iam.js (hunk 1 above)
Comment (mayankbharadwaj) [reply aws_promise/iam.js:8]: Issue was with const initializer as const variable needs to be assigned with some value.

**Review by RajkumarShivage — COMMENTED** [review]
LGTM

**Review by bhargavapratik — APPROVED** [review]
LGTM

**Review by RajkumarShivage — APPROVED** [review]
LGTM

**Pull request merged**